django_airavata/apps/api/data_products_helper.py

```diff
@@ -129,18 +130,65 @@ def delete_dir(request, path):
     return datastore.delete_dir(request.user.username, path)
 
 
+def delete_user_file(request, path):
```

Comment:
As much as possible, data_products_helper functions should work on DataProduct instances. There already is a `delete(request, data_product)` and there is an API to access it. See api/views.py, function delete_file (https://github.com/apache/airavata-django-portal/blob/develop/django_airavata/apps/api/views.py#L1009).

Reply:
Yes, I saw that too. But, I was trying to make this view generic for both files and directories. So, that I could make the request call from front end for both files and directories the same